tests/controls/windowtest.cpp

```diff
@@ -148,6 +195,21 @@ TEST_CASE_METHOD(WindowTestCase, "Window::Mouse", "[window]")
 
     CHECK(m_window->GetCursor().IsOk());
 
+#ifndef __WXGTK__
+    wxCursor resCursor1( "horse" );
+    CHECK( resCursor1.IsOk() );
+#else
+    wxCursor rresCursor1( (const char *) horse, 32, 32, 16, 0, nullptr, wxWHITE, wxBLACK );
+    CHECK( rresCursor1.IsOk() );
+#endif
```

Comment:
Could we get at least some comments about what's going on here? Why is wxGTK handled specially? Surely creating cursors from the embedded data should work in all ports?

Reply:
@vadz ,
Do you want me to quote the documentation here?

Reply:
@vadz ,
I added some comments explaining the difference between X/GTK and Windows/OSX.

Let me know if this is not enough.

The code in the test can be used as is in the documentation instead of the current one.